backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/AddDiskCommand.java

```diff
@@ -846,10 +847,18 @@ private void plugDiskToVmIfNeeded() {
     }
 
     protected boolean setAndValidateDiskProfiles() {
+        if (shouldSkipDiskProfilesValidation()) {
```

Comment:
please check if you can move this check to DiskProfileHelper so it will also cover move/copy and import operations

Reply:
For import we don't need it (it requires the admin role `VmImporterExporter` - right?), which has the attach disk permission. What is the role for move/copy disks? If it's an admin role, it most likely have the attach disk permission as well.

Reply:
look at the broader scope, not just the bug where you fail to suspend or take a snapshot when you lack permissions for attaching a disk profile, this change is broader - it prevents setting memory dump/metadata disk with disk profiles. as I wrote above to @bennyz , I think this is the right approach because those are not really virtual disks. so even if you have the required permissions to set disk profiles in some scenario, why would you want to do that? if you have a good reason to do it in these cases, that might be a good reason to do it when creating the disks - but I can't see a reason for that